src/ReactiveUI/CreatesCommandBinding.cs

```diff
@@ -4,15 +4,20 @@
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using System.Reflection;
-using System.Windows;
 using System.Windows.Input;
 
 #if NETFX_CORE
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Input;
+#elif WPF
+using System.Windows;
 #endif
 
+#if WPF
+namespace ReactiveUI.Wpf
```

Comment:
The `RoutedEventArgs` type is in the WPF assemblies, so including the `Click` event as a default event won't work unless we use the `EventArgs` type.